src/hotspot/cpu/ppc/register_ppc.hpp

```diff
@@ -100,6 +100,7 @@ class Register {
   constexpr bool is_valid()       const { return ( 0 <= _encoding && _encoding <  number_of_registers); }
   constexpr bool is_volatile()    const { return ( 0 <= _encoding && _encoding <= 13 ); }
   constexpr bool is_nonvolatile() const { return (14 <= _encoding && _encoding <= 31 ); }
+  constexpr bool is_nonvolatile_accross_preemption() const { return (_encoding == 22) || (_encoding == 31 ); }
```

Comment:
I'd appreciate a short comment before your new function. The other functions in this section are defined by the ABI. This one is implementation specific to keep values in `TemplateInterpreterGenerator::generate_native_entry` alive after preemption and resuming.